src/analysis/inner_product_space/symmetric.lean

```diff
@@ -152,4 +152,42 @@ end
 
 end complex
 
+section
+
+/-- polarization identity for symmetric linear maps -/
```

Comment:
This should include a reference to the existing lemma for `complex` that doesn't need `hT`.

Reply:
what do you mean?

Reply:
I mean that you should include a sentense like "see also `the_lemma_name` for a version for `complex` without the `symmetric` assumption"

Reply:
done